src/components/Verse/VerseText.tsx

```diff
@@ -21,31 +20,10 @@ type VerseTextProps = {
   isHighlighted?: boolean;
 };
 
-const READING_MODE_ROOT_MARGIN = '-10% 0px -85% 0px';
-const DEFAULT_ROOT_MARGIN = '-23% 0px -72% 0px';
-const OBSERVER_THRESHOLD = 0.01;
-
+const OBSERVER_NAME = 'quranReaderObserver';
```

Comment:
should this be imported instead of redefined? and maybe it should be ID instead of name

`QURAN_READER_OBSERVER_ID`